community/config/eggs/creating_a_custom_egg.md

```diff
@@ -30,6 +30,29 @@ can be tweaked per-server if needed).
 _Docker images must be specifically designed to work with Pterodactyl Panel._ You should read more about that in
 our [Creating a Docker Image](/community/config/eggs/creating_a_custom_image.md) guide.
 
+## The Pterodactyl Install Procces
+
+::: warning
+Please be aware of how the pterodactyl install proces works!
+:::
+
+```
+1. Spin up install container
+    Creates a new container using an install image that's run as root.
+    Uses a volume mount on `/mnt/server` for the server files, which is the working directory during installation.
+    The volume will be later mounted as `/home/container` for the server container. Any files outside of `/mnt/server` will be gone after installation.
+    Install script can pull files or set up all that is needed to run the server, such as writing files, directories or compiling apps.
```

Comment:
I'd change this to "The installation script can set up everything that's required to run the server, such as writing files, creating directories, or compiling apps."